specs/deposits.md

```diff
@@ -144,7 +144,9 @@ follows:
 - `context.value` set to `sendValue`
 
 No gas is bought on L2 and no refund is provided. The gas used for the deposit is subtracted from
-the gas pool on L2.
+the gas pool on L2. Gas usage exactly matches other transaction types (including intrinsic gas).
```

Comment:
is 'gas pool' a commonly used term? I think 'block gas limit' is more intuitive.